Formula/broot.rb

```diff
@@ -19,6 +20,24 @@ class Broot < Formula
 
   def install
     system "cargo", "install", *std_cargo_args
+
+    # Replace man page "#version" and "#date" based on logic in release.sh
+    build_date = Utils.safe_popen_read("date -u +'%Y/%m/%d' 2> /dev/null").chomp
+    inreplace "man/page" do |s|
+      s.gsub! "#version", version
+      s.gsub! "#date", build_date
+    end
```

Comment:
We're downloading a release. Shouldn't this be done already?

Reply:
It looks like the man page is only added into pre-built GitHub zip archive.
I think Homebrew uses source code for release (`broot-<version>.tar.gz`), which only has the unmodified `./man/page` file.